CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift

```diff
@@ -79,10 +79,9 @@ class OnboardingViewController: UIViewController {
 
     private func configureBasicNavigation() {
         nextButton.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
-        nextButton.accessibilityValue = NSLocalizedStringPreferredFormat("ginicapture.onboarding.next",
-                                                                         comment: "Next button")
         nextButton.configure(with: GiniConfiguration.shared.primaryButtonConfiguration)
         nextButton.addTarget(self, action: #selector(nextPage), for: .touchUpInside)
+        configureNextButton()
 
```

Comment:
[Suggestion] @ValentinaIancu-Gini could you please remove extra line here?

Reply:
I think is better to have an extra line to visualize when the configuration for the next button is finished and when the skip button is started.